labellab-flask/requirements.txt

```diff
@@ -20,6 +20,7 @@ Flask==1.1.1
 Flask-Bcrypt==0.7.1
```

Comment:
Do we need 72 dependencies?? some versions doesn't work in this list.

Reply:
Most of them are the additional dependencies that come with the packages we have used, I haven't added any extra dependency that we haven't used. Yes, i noticed some of the versions were failing and they have been changed in the other pull requests.